### DISCUSSION TITLE
PTZ-Support

### DISCUSSION THREAD

**Discussion by gerd2005** [koush/scrypted#1250] [Q&A] [answered]
Hello,

Does Scrypted/Homekit support PTZ cameras such as these?

https://m.reolink.com/de/product/rlc-823a/?attribute_pa_version=1-pack&lang=de&gad_source=1&gclid=Cj0KCQiAtOmsBhCnARIsAGPa5yaM4Yc9CZXt77-gyV_mT2kz_h6gjwe_oB3InPFCbgXQw1YHu820jKMaAkOtEALw_wcB

Best regards
Gerd

**Comment by koush** [comment, marked as answer]
reolink plugin supports ptz now

**Reply by AKaiserRoll** [reply]
I am also wondering how to control the camera in the Home app. I see that it is supported in scrypted, but I don't see a way to control it when I pull up the camera stream.

**Comment by gerd2005** [comment]
Ok, I see, thank you. But where are the PTZ-Settings in HomeKit?
